samples/react-sample-app/babel.config.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Let's check the other places as well

Reply:
Resolved at https://github.com/wso2-extensions/identity-verification-onfido/pull/17